dashboard/src/actions/overviewActions.js

```diff
@@ -13,16 +12,13 @@ import { uriTemplate } from "../utils/helper";
 
 export const getDatasets = () => async (dispatch, getState) => {
   const alreadyRendered = getState().overview.loadingDone;
-  const datasetType = getState().comparison.datasetType;
-
-  const loggedIn = Cookies.get("isLoggedIn");
   try {
     if (alreadyRendered) {
       dispatch({ type: TYPES.LOADING });
     }
     const params = new URLSearchParams();
 
-    addParams(params, loggedIn, datasetType);
+    addParams(params);
```

Comment:
How does the `?mine` filter get added on the Overview page now??

I think this could have been solved by hardcoding "my datasets" `datasetType` in the call here, (assuming this is really used [only] for the Overview page?) whereas now I believe we're unconditionally getting all visible datasets?

Reply:
Oops! I forgot the main part. 

Updated the code.